besskge/sharding.py

```diff
@@ -341,14 +344,15 @@ def create_from_queries(
         :param query_mode:
             "hr" for (h,r,?) queries, "rt" for (?,r,t) queries.
         :param ground_truth: shape: (n_query,)
-            If know, the global ID of the ground truth tail/head.
+            If known, the global ID of the ground truth tail/head.
         :param negative: shape: (N, n_negative)
             Global IDs of negative entities to score against each query,
-            query-specific (N=n_query) or same for all queries (N=1).
-            Defaults to None (i.e. score queries against all entities in the graph).
+            query-specific (N=n_query) or the same for all queries (N=1).
+            Default: None (namely the score queries against all entities in the
```

Comment:
The additional "the" here doesn't sound quite right to me, I was using "score" as a verb (with "queries" as direct object). Maybe we can rephrase "(namely score each query against all entities...)"

Reply:
This is my proposal:
```
        :param negative: shape: (N, n_negative)
            Global IDs of negative entities to score against each query. This
            can be query-specific (N=n_query) or the same for all queries (N=1).
            Default: None (namely score each query against all entities in the
            graph).
        :param negative_type:
            Score each query only against entities of a specific type. Default:
            None (namely score each query against entities of any type).
```